python/jiminy_py/src/jiminy_py/log.py

```diff
@@ -239,47 +299,79 @@ def extract_trajectory_from_log(log_data: Dict[str, Any],
               field "evolution_robot" and it is a list of State object. The
               other fields are additional information.
     """
+    return extract_trajectories_from_log(log_data, [robot])[robot.name]
```

Comment:
Does this really work if `robot`  is None ? I guess it does because `robot` is updated through `extract_trajectories_from_log` - then I would suggest writing it in two lines (like `build_robot_from_log`) for readability (possibly doing the same thing that @duburcqa suggested for this function).

Reply:
The method extract_trajectories_from_log is now calling extract_trajectory_from_log, so it should have disappeared.